docs/dev/dev_setup.rst

```diff
@@ -203,6 +203,51 @@ with linters and other code analysis tools.
 Creating a Workspace
 -------------------------------------------------------------------------------
 
+PUDL Environment Variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+PUDL needs to know where to store its big piles of inputs and outputs.
+The ``PUDL_OUTPUT`` and ``PUDL_CACHE`` environment variables lets PUDL know where
+all this stuff should go. We call this a "PUDL workspace".
+
+First, create a directory to store local caches of raw PUDL data. Then set
+``PUDL_CACHE`` variable to the path of the new directory:
+
+.. code-block:: console
+
+    mamba env config vars set PUDL_CACHE=</path/to/cache/directory/>
+    mamba activate pudl-dev
+
+Next, create a directory to store the outputs of the PUDL ETL. Then set the
+``PUDL_OUTPUT`` variable to the path of the new directory:
+
+.. code-block:: console
+
+    mamba env config vars set PUDL_OUTPUT=</path/to/output/directory/>
+    mamba activate pudl-dev
+
+**Make sure you create separate directories for these environment variables!
+It is recommended you create these directories outside of the pudl repository
+directory so the inputs and outputs are not tracked in git.**
+
+PUDL Setup (old setup method)
```

Comment:
Because the sections are titled "PUDL Environment Variables" and "PUDL Setup (old setup method)" it's unclear whether they are independent of one another. Is the first part the "new setup method"? Should we just be paying attention to this first section, or do we still need to use the old setup method too? Would be nice to clarify the titles or add a sentence about this.

Reply:
I updated the section title to be clearer and I included a note saying you can skip the new setup if you used the old setup.